devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -90,11 +95,23 @@ def parseAttrMessage(description) {
 
 def getButtonEvent(descMap) {
 	if (descMap.commandInt == 1) {
-		getButtonResult("press")
-	}
-	else if (descMap.commandInt == 0) {
-		def button = buttonMap[device.getDataValue("model")][descMap.sourceEndpoint]
-		getButtonResult("release", button)
+		if (isBSM300() || isMSM300() || isSBM300ZB1() || isSBM300ZB2() || isSBM300ZB3()) {
```

Comment:
this look much better so far, but could you just combine all of these into one `isShinaButton` function or something like that, since they're never called separately?